src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.props

```diff
@@ -3,18 +3,6 @@
     Properties and tasks supporting Razor MSBuild integration
   -->
 
-  <!--
-    Razor defines two primary targets:
-      'RazorGenerate' - which updates generated code
-      'RazorCompile' - compiles an assembly from generated code
-
-    Use these properties to attach behavior to the corresponding target.
-  -->
-  <PropertyGroup>
-    <RazorGenerateDependsOn>RazorResolveGenerateInputs;RazorCoreGenerate</RazorGenerateDependsOn>
-    <RazorCompileDependsOn>RazorGenerate;RazorCoreCompile</RazorCompileDependsOn>
-  </PropertyGroup>
```

Comment:
Moved these into the .targets. That's the style that the .net targets follow. With Before/After targets nowadays, this isn't super meaningful for anyone but us anyways.